tests/ct_connection_vhost_ports_with_ping_icmp.py

```diff
@@ -0,0 +1,165 @@
+# Copyright (c) 2022 Intel Corporation.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at:
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+DPDK Connection Tracking with vhost Port for ping test
+"""
+
+# in-built module imports
+import time
+import sys
+
+# Unittest related imports
+import unittest
+
+# ptf related imports
+import ptf
+import ptf.dataplane as dataplane
+from ptf.base_tests import BaseTest
+from ptf.testutils import *
+from ptf import config
+
+# scapy related imports
+from scapy.packet import *
+from scapy.fields import *
+from scapy.all import *
+
+
+# framework related imports
+import common.utils.p4rtctl_utils as p4rt_ctl
+import common.utils.log as log
+import common.utils.test_utils as test_utils
+from common.utils.config_file_utils import get_config_dict, get_gnmi_params_simple, get_interface_ipv4_dict
+from common.utils.gnmi_ctl_utils import gnmi_ctl_set_and_verify, gnmi_set_params, ip_set_ipv4
+from common.lib.telnet_connection import connectionManager
+
+class Connection_Track(BaseTest):
+
+    def setUp(self):
+        BaseTest.setUp(self)
+        self.result = unittest.TestResult()
+        config["relax"] = True # for verify_packets to ignore other packets received at the interface
+        
+        test_params = test_params_get()
+        config_json = test_params['config_json']
+        self.dataplane = ptf.dataplane_instance
+        ptf.dataplane_instance = ptf.dataplane.DataPlane(config)
```

Comment:
In this case, only ping is used, no dataplane traffic involved. Suggest to remove line 57-58.